### PR TITLE
Rolling back change of task lib in AzurePowerShellV4

### PR DESCRIPTION
**Pull request by 20shivangi**

### CODE REVIEW

File: Tasks/AzurePowerShellV4/task.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "version": {
         "Major": 4,
         "Minor": 0,
-        "Patch": 5
+        "Patch": 6
```
Comment (niadak) [inline Tasks/AzurePowerShellV4/task.json]: I am assuming we will update the patch version in master to 7 before M152 cut off.

Comment (20shivangi) [reply Tasks/AzurePowerShellV4/task.json]: yes, I will do

File: Tasks/AzurePowerShellV4/azurepowershell.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import fs = require('fs');
 import path = require('path');
 import os = require('os');
-import tl = require('azure-pipelines-task-lib/task');
-import tr = require('azure-pipelines-task-lib/toolrunner');
+import tl = require('vsts-task-lib/task');
```
Comment (kmkumaran) [inline Tasks/AzurePowerShellV4/azurepowershell.ts]: Please test in both Windows and Linux agents.

Comment (20shivangi) [reply Tasks/AzurePowerShellV4/azurepowershell.ts]: Tested in both agents.

**Pull request merged**